lib/crypto/crc.toit

```diff
@@ -210,8 +215,730 @@ class Crc extends Checksum:
   /**
   See $super.
 
-  Returns the checksum as a width/4 element byte array in little-endian order.
+  Returns the checksum as a width/4 element byte array in the endian order that
+    corresponds to the constructor used.
   */
   get -> ByteArray:
     checksum := sum_ ^ xor_result
-    return ByteArray (width + 7) / 8: (checksum >> (8 * it)) & 0xff
+    result := ByteArray (width + 7) >> 3
+    if little_endian:
+      binary.LITTLE_ENDIAN.put_uint result result.size 0 checksum
+    else:
+      binary.BIG_ENDIAN.put_uint result result.size 0 checksum
+    return result
+
+/**
+Computes the CRC-16/CCITT-FALSE checksum of the given $data.
+
+The $data must be a string or byte array.
+Returns the checksum as a 16-bit integer.
+*/
+crc_16_ccitt_false data from/int=0 to/int=data.size -> int:
```

Comment:
I would drop the `from` and `to` here and for all the others. We have slices for that.

If kept, they should be named arguments. `crc_16_ccitt_false data 3` wouldn't read nicely or convey what it does.